docs/reference/operations/create_transfers.md

```diff
@@ -224,6 +255,18 @@ The transfer was not created.
 [`flags.post_pending_transfer`](../transfers.md#flagspost_pending_transfer) and
 [`flags.void_pending_transfer`](../transfers.md#flagsvoid_pending_transfer).
 
+### `cannot_balance_debit`
```

Comment:
I think that `cannot_balance_debit` reads more like "cannot balance the debit side", whereas `balancing_debit` reads more like "a debit that balances the credit side".

It would be better, I think, also to bring the mutually exclusive part into the error message directly, and perhaps distinguish between the flags further. For example:

`balancing_debit_is_exclusive_with_post_pending_transfer`
`balancing_debit_is_exclusive_with_void_pending_transfer`

Reply:
:thinking: I was avoiding that for two reasons:
- It would require 4 new error codes. And it requires an extra 2 codes for each incompatible flag we add in the future. I'm worried we could end up in an n² situation.
- With the error naming, there isn't an obvious precedence to use in general. I'm tempted to go the other extreme: just have a catch-all `incompatible_flags` error (we could use that for the post+void combination too), and let the client sort it out (since incompatible flags are always an application/dev bug).

Alternatively, maybe a naming convention that is more consistent? These flags are in the same vein as `cannot_post_and_void_pending_transfer` and `pending_transfer_cannot_post_or_void_another`, but use different conventions. A catch-all is simplest, though.

Reply:
Agreed, also on the catch-all.

Let's do `mutually_exclusive_flags`.

Reply:
Or `flags_mutually_exclusive` to follow MSB.

Reply:
Sounds good; and I'll rename `Account`'s `mutually_exclusive_flags` to `flags_are_mutually_exclusive` so they match.